conf/web.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-exec uwsgi --http :${PORT:-8000} --wsgi-file /app/recommendation/wsgi.py --master
+exec uwsgi --wsgi-disable-file-wrapper --http :${PORT:-8000} --wsgi-file /app/recommendation/wsgi.py --master
```

Comment:
This [is necessary](https://www.python.org/dev/peps/pep-0333/#optional-platform-specific-file-handling) to let uwsgi return a streamed `ByteIO` object.